resources/dicts/patrols/general/medcat.json

```diff
@@ -700,15 +728,24 @@
         "platonic"
     ],
     "intro_text": "p_l leads app1 through c_n's territory, enjoying the nice day. They glance back, and realize app1 looks like they want to ask them a question.",
-    "decline_text": "Just as app1 opens their mouth to speak, p_l spots a Clanmate trying to remove a thorn from their paw over their shoulder, and they hurry away to help.",
+    "decline_text": "Just as app1 opens their mouth to speak, p_l spots a Clanmate trying to remove a thorn from their paw over app1's shoulder, and they hurry away to help.",
     "chance_of_success": 50,
     "exp": 20,
     "success_text": [
         "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 tentatively explains that they were curious about what it's like to get a vision from StarClan. p_l ponders how to explain it, and settles for a vague description and iterating that it's different for every cat who has one. app1 seems a bit confused, but grateful.",
-        "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 studiously looks at their paws and mutters that they don't want to talk about it, seemingly having thought better of it. p_l gently reassures them that it's okay to ask questions, and though app1 doesn't ask, they seem more chipper, and they're a big help anyways."
+        "p_l sits down and motions for app1 to join them, looking at them inquisitively. app1 studiously looks at their paws and mutters that they don't want to talk about it, seemingly having thought better of it. p_l gently reassures them that it's okay to ask questions, and though app1 doesn't ask, they seem more chipper, and they're a big help anyways.",
+        "s_c sits down and motions for app1 to join them, looking at them inquisitively. app1 tentatively explains that they were curious about what it's like to get a vision from StarClan. s_c is probably the best cat they could ask that question, and the older cat takes pride in the way app1 listens to them talk with wonder shining in their eyes."
```

Comment:
last bit might read better like this:
`s_c is probably the best cat they could have asked that question to. The older cat takes pride in the way app1 listens to them talk, wonder shining in their eyes.`